week-1/interpret/time-format.js

```diff
@@ -14,16 +14,22 @@ console.log(result);
 // For the piece of code above, read the code and then answer the following questions
 
 // a) How many variable declarations are there in this program?
+// Ans. 7
 
 // b) How many function calls are there?
+// Ans. None
 
 // c) Using documentation on MDN, explain what the expression movieLength % 60 represents
+//Ans. This expression is providing us with the remainder of any seconds which are less than 60.
```

Comment:
Could you add some more clarity to this answer, I'm not sure I understand what "less than 60" is referring to

Reply:
I have tried adding some more explanation